README.md

```diff
@@ -51,6 +46,47 @@
 
 Aries Framework JavaScript is a framework written in TypeScript for building **SSI Agents and DIDComm services** that aims to be **compliant and interoperable** with the standards defined in the [Aries RFCs](https://github.com/hyperledger/aries-rfcs).
 
+## Packages
+
+<table>
+  <tr>
+    <th><b>Package</b></th>
+    <th><b>Version</b></th>
+  </tr>
+  <tr>
+    <td>Core</td>
```

Comment:
Maybe we can be more explicit:

`@aries-framework/core` instead of Core, etc..?